packages/base/Tag/src/Tag/Tag.tsx

```diff
@@ -16,31 +16,50 @@ import { twMerge } from 'tailwind-merge'
 import { classByVariant } from './styles'
 
 export type Variant = 'light-grey' | 'blue' | 'green' | 'yellow' | 'red'
-
 export type DivOrAnchorProps = AnchorHTMLAttributes<HTMLAnchorElement> &
   HTMLAttributes<HTMLDivElement>
 
 export interface Props extends BaseProps, TextLabelProps, DivOrAnchorProps {
-  /** The component used for the root node. Either a string to use a DOM element or a component. */
   as?: ElementType
-  /** Text content of the `Tag` component */
   children: ReactNode
-  /** Specify the icon which should be rendered inside Tag */
   icon?: ReactElement
-  /** Defines if `Tag` is disabled */
   disabled?: boolean
-  /** A callback which is invoked after remove `Icon` is clicked
-   *
-   * Please note that specifying this callback automatically adds remove `Icon` as children of the `Tag`
-   */
   onDelete?: () => void
-  /** Variant of the `Tag` */
   variant?: Variant
-  /** ReactNode rendered after label */
   endAdornment?: ReactNode
 }
 
-export const Tag = forwardRef<HTMLDivElement, Props>(function Tag(props, ref) {
+type IconProps = {
+  className?: string
+  color?: string
+}
+
+const DeleteIcon = ({ onClick }: { onClick: (event: MouseEvent) => void }) => (
+  <span
+    aria-label='delete icon'
+    role='button'
+    className='w-min h-min flex items-center cursor-pointer -ml-2 mr-2'
+    onClick={onClick}
+  >
+    <CloseMinor16 />
+  </span>
+)
+
+const cloneIcon = (icon: ReactElement<IconProps>, disabled?: boolean) => {
+  if (!icon || !React.isValidElement(icon)) {
+    return null
+  }
+
+  return React.cloneElement(icon, {
+    color: disabled ? 'grey' : icon.props.color || 'darkGrey',
+    className: twMerge(
+      'flex items-center -mr-1 ml-3',
+      (icon.props as BaseProps).className
+    ),
+  })
+}
+
+const Tag = forwardRef<HTMLDivElement, Props>(function Tag(props, ref) {
```

Comment:
We are not exporting `Tag` anymore, is this intentional?
```suggestion
export const Tag = forwardRef<HTMLDivElement, Props>(function Tag(props, ref) {
```

Reply:
✅